src/app/core/components/navigation/navigation.component.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  <button [ngClass]="{'active': activeTab === activeRouteNavigationEnum.CharactersRoute}" (click)="toCharacters()">Characters</button> /
```

Comment:
also you can do the same with

`[class.active]="activeTab === activeRouteNavigationEnum.CharactersRoute"`